### ISSUE TITLE
[bug] Connecting to WalletConnect-based wallets in Chrome on iOS returns users to a blank tab

### ISSUE THREAD

**Issue by markdalgleish** [rainbow-me/rainbowkit#524]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### RainbowKit Version

0.3.4

### wagmi Version

0.4.10

### Current Behavior

When connecting to a WalletConnect-based wallet in Chrome on iOS, once you've connected and been returned to the browser, you're presented with an empty tab at `about:blank`. You then need to manually close to return to the original page that triggered the connection.

### Expected Behavior

You should go back to the same page you were on previously.

### Steps To Reproduce

1) Go to https://rainbowkit-example.vercel.app in Chrome on iOS.
2) Tap "Connect Wallet".
3) Tap "Rainbow" -> Goes to Rainbow app.
4) Tap "Connect" -> Goes back to browser.
5) See `about:blank` tab.
6) Close tab to return to original page.

### Link to Minimal Reproducible Example (CodeSandbox, StackBlitz, etc.)

https://rainbowkit-example.vercel.app

### Anything else?

It's likely that this affects other browsers, but so far I've only attempted to reproduce this in Chrome.

**Comment by markdalgleish** [comment]
I tried to find a fix for this and these two approaches seem to work:
- For the wallet options on mobile, render links instead of buttons. While it seems simple on the surface, supporting this would require a decent change to our architecture. Or;
- Use `window.location.href = ...` rather than `window.open` when navigating to the app's universal link. This is a much simpler fix but we'd need to validate that this doesn't have unintended UX impacts for users who don't have the app installed.

**Comment by intergalacticspacehighway** [comment]
+1 for window.location.href. It also solves a weird unrelated but similar issue.


The below is opened in "in-app twitter" browser.

### with window.open


https://user-images.githubusercontent.com/23293248/175381493-30271a16-e18f-445b-8cf8-946b1a2c236c.MP4

notice the window becomes blank. Snippet used for testing (added with safari mac devtool)

```js
$0.addEventListener('click',()=> {
   window.open("https://rnbwapp.com")
});
```

### with window.location.href
https://user-images.githubusercontent.com/23293248/175381532-58612d05-2358-4a48-9da9-2b99c1ebbc41.MP4

window doesn't become blank.

```js
$0.addEventListener('click',()=> {
   window.location.href = "https://rnbwapp.com"
});
```

**Comment by markdalgleish** [comment]
I've opened a PR with a fix for this: https://github.com/rainbow-me/rainbowkit/pull/540

I found that creating a link in memory with `target` set to `"_blank"` and then calling `.click()` on it also fixes the issue, while preserving the behaviour that users without the app installed get a new tab to prompt a download rather than replacing the current page.

**Comment by markdalgleish** [comment]
Fixed in [RainbowKit v0.3.6](https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Frainbowkit%400.3.6).
